### PR TITLE
Add filter is_twelve_month , is_most_recent in global_data and

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
## Changes
-Add filter is_twelve_month , is_most_recent in global_data and reginal_data

* Detailed list or prose of changes
* Breaking changes
* Changes to configurations

Mention related users here if any.

## This PR doesn't introduce any:

- [ ] temporary files, auto-generated files or secret keys
- [ ] n+1 queries
- [ ] flake8 issues
- [ ] `print`
- [ ] typos
- [ ] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)

### CODE REVIEW

File: apps/visualization/filters.py [hunk 1]
```diff
@@ -53,15 +63,81 @@ class EpiDataGlobalFilter():
...
+            return queryset.filter(Q(region='Global'))
+        return queryset.filter(~Q(region='Global'))
+
+    def filter_is_twelve_month(self, queryset):
+        if self.is_twelve_month:
+            return queryset.order_by('-context_date')
+
+    def filter_is_regional_chart(self, queryset):
+        if self.is_regional_chart:
+            most_recent_reginal_data = queryset.filter(most_recent=True).exclude(region='Global').values('region').annotate(
```
Comment (Rup-Narayan-Rajbanshi) [inline apps/visualization/filters.py]: most_recent_reginal_data typo

File: apps/visualization/filters.py [hunk 2]
```diff
@@ -53,15 +63,81 @@ class EpiDataGlobalFilter():
...
+                recent_context_date=Max('context_date'),
+                max_context_indicator_value=Max('context_indicator_value')
+            ).order_by('region')
+            if most_recent_reginal_data:
```
Comment (Rup-Narayan-Rajbanshi) [inline apps/visualization/filters.py]: most_recent_reginal_data typo

File: apps/visualization/filters.py [hunk 3]
```diff
@@ -53,15 +63,81 @@ class EpiDataGlobalFilter():
...
-        if self.is_global == False:
-            return queryset.filter(~Q(region='Global'))
-        return queryset.filter(Q(region='Global'))
+        if self.is_global:
+            return queryset.filter(Q(region='Global'))
+        return queryset.filter(~Q(region='Global'))
+
+    def filter_is_twelve_month(self, queryset):
+        if self.is_twelve_month:
+            return queryset.order_by('-context_date')
```
Comment (Rup-Narayan-Rajbanshi) [inline apps/visualization/filters.py:75]: Is this logic enough for 12 month data ?

Comment (bimal125) [reply apps/visualization/filters.py:75]: @Rup-Narayan-Rajbanshi  This table holds data per month, client will apply limit =12 offset =0 and get 12 months recode

**Review by Rup-Narayan-Rajbanshi — COMMENTED** [review]
Looks good.

**Pull request merged**